packages/playground/src/components/reserve_action_btn.vue

```diff
@@ -0,0 +1,149 @@
+<template>
+  <container>
+    <v-dialog v-model="openUnreserveDialog" max-width="600" class="custom-dialog">
+      <v-card>
+        <v-card-title class="pt-4">
+          <span style="display: flex; align-items: center">
+            <v-icon color="warning" style="margin-right: 8px">mdi-alert</v-icon>
+            Are you sure you want to unreserve this dedicated node?
+          </span>
+        </v-card-title>
+        <v-divider style="color: #1aa18f" />
```

Comment:
What is this color is used for?

Reply:
![Screenshot from 2023-10-11 14-30-36](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/54944491/aebc2061-4ce4-4310-ad8c-50ad0362ac37)

Reply:
it could be our primary color.

Reply:
yes but if i typed `primary` it doesn't reflect, that's why i added it like that.